stoservice/src/main/java/com/tutrit/stoservice/dispatcher/RequestDispatcher.java

```diff
@@ -1,22 +1,27 @@
-package com.tutrit.stoservice.controller;
+package com.tutrit.stoservice.dispatcher;
 
 
+import com.tutrit.stoservice.controller.*;
+import com.tutrit.stoservice.utility.ParseCommand;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import static com.tutrit.stoservice.context.ApplicationContext.get;
+
 public class RequestDispatcher {
 
     public static Logger logger = LoggerFactory.getLogger("Dispatcher");
 
     public void doDispatch(String userInput) {
-        Request request = new Request(userInput);
+        Request request = new Request(get(ParseCommand.class).getCommand(userInput));
```

Comment:
I don't think it's necessary to take ParseCommand.class out of context. Methods of ParseCommand.class can be made static